### ISSUE TITLE
Labels are not used

### ISSUE THREAD

**Issue by felixholub** [jmboehm/RegressionTables.jl#30]
Recently a change was made regarding the type of dictionary accepted as labels, now a `Dict{Symbol,String}` is required. But any value provided now is just ignored. For example:
```
using RegressionTables, DataFrames, FixedEffectModels, RDatasets

df = dataset("datasets", "iris")
df[:SpeciesDummy] = categorical(df[:Species])

rr1 = reg(df, @model(SepalLength ~ SepalWidth   , fe = SpeciesDummy))

regtable(rr1; renderSettings = asciiOutput(), 
        labels = Dict(:SpeciesDummy => "My categorical variable"))

```
yields:
```
--------------------------
               SepalLength
               -----------
                       (1)
--------------------------
SepalWidth        0.804***
                   (0.106)
--------------------------
SpeciesDummy           Yes
--------------------------
Estimator              OLS
--------------------------
N                      150
R2                   0.726
--------------------------
```

**Comment by jmboehm** [comment]
Thanks -- yeah, looks like something went wrong in the most recent fix. I'll sort it out in the next couple of days.

**Comment by jmboehm** [comment]
Should be fixed by #32 . Note that the `label` dict now takes strings as keys, not symbols.
